opm/parser/eclipse/EclipseState/EclipseState.hpp

```diff
@@ -70,33 +72,15 @@ namespace Opm {
         std::shared_ptr< const SimulationConfig > getSimulationConfig() const;
         std::shared_ptr< const EclipseGrid > getEclipseGrid() const;
         std::shared_ptr< EclipseGrid > getEclipseGridCopy() const;
-        bool hasPhase(enum Phase::PhaseEnum phase) const;
         std::string getTitle() const;
-        bool supportsGridProperty(const std::string& keyword, int enabledTypes=AllProperties) const;
-
-        std::shared_ptr<const GridProperty<int> > getRegion( const DeckItem& regionItem ) const;
-        std::shared_ptr<const GridProperty<int> > getDefaultRegion() const;
-        std::shared_ptr<const GridProperty<int> > getIntGridProperty( const std::string& keyword ) const;
-        std::shared_ptr<const GridProperty<double> > getDoubleGridProperty( const std::string& keyword ) const;
-        bool hasDeckIntGridProperty(const std::string& keyword) const;
-        bool hasDeckDoubleGridProperty(const std::string& keyword) const;
-
-        bool hasIntGridProperty(const std::string& keyword) const __attribute__((deprecated("use hasDeckIntGridProperty() instead")))
-        { return hasDeckIntGridProperty(keyword); }
-        bool hasDoubleGridProperty(const std::string& keyword) const __attribute__((deprecated("use hasDeckDoubleGridProperty() instead")))
-        { return hasDeckDoubleGridProperty(keyword); }
-
-        void loadGridPropertyFromDeckKeyword(std::shared_ptr<const Box> inputBox,
-                                             const DeckKeyword& deckKeyword,
-                                             int enabledTypes = AllProperties);
 
         std::shared_ptr<const FaultCollection> getFaults() const;
         std::shared_ptr<const TransMult> getTransMult() const;
         std::shared_ptr<const NNC> getNNC() const;
         bool hasNNC() const;
 
+        const Eclipse3DProperties& getEclipseProperties() const;
```

Comment:
> Sorry: this should be getEclipse3DProperties() - the room for confusion regarding names consisting of Eclipse and properties is endless.

actually I have to concure: I'm already confused. Aren't grids always three dimensional and thus all properties?